measures/iama.yaml

```diff
@@ -1,10 +1,10 @@
-systemcard_path: .assessments[]             # append iama.yaml to 'assessments' list in system card 
-schema_version: v_1.0.0
+systemcard_path: .assessments[]               # append iama.yaml to 'assessments' list in system card 
+schema_version: iama_v_1.0.0
```

Comment:
Ah, I don't think we  should have an iama specific schema. This should refer to the instrument schema version.